### ISSUE TITLE
Encouraging Spell is not a valid choice for Skald level 1 bonus feat.

### ISSUE THREAD

**Issue by CaptOcto** [Vek17/TabletopTweaks-Base#658]
As title, seems out of place considering the spell list and that every other metamagic feat (even TTT one) are viable choices.

**Comment by Vek17** [comment]
Encouraging spell can't be taken at level 1 anyway dye to prerequisites

**Comment by CaptOcto** [comment]
Yes this is via a dip and all other metamagics were there even Selective Spell.

**Comment by CaptOcto** [comment]
Sorry you were right, probably working as intended and my character just didn't meet the Persuasion requirement!
